optuna/samplers/_tpe/parzen_estimator.py

```diff
@@ -61,7 +60,7 @@ def __init__(
 
         if len(transformed_observations) == 0:
             weights = np.array([1.0])
-        elif parameters.consider_prior:
+        else:
             assert parameters.prior_weight is not None
             weights = np.append(weights, [parameters.prior_weight])
```

Comment:
As `prior_weight` is already guaranteed to be not None from the earlier line, we can simply code like this:

```python
        weights = np.append(weights, [parameters.prior_weight])
```